doc/source/user_manual/configuration/configurationdata.rst

```diff
@@ -22,7 +22,8 @@ Details for configuring connectors are described in :ref:`output` and :ref:`inpu
 General information about the configuration of the pipeline can be found in :ref:`pipeline_config` .
 
 It is possible to use environment variables in all configuration and rules files in all places.
-Environment variables have to be set in uppercase. Lowercase variables are ignored.
+Environment variables have to be set in uppercase. Lowercase variables are ignored. Forbidden
+variable names are: :code:`["LOGPREP_LIST"]`
```

Comment:
We never really discussed a pattern for the envs but should we maybe rename this to something like `LOGPREP_LISTCOMPARISON_LIST`? Ah though I am realizing now that this would be a breaking change...

Reply:
yes... it would be a breaking change... and I would not touch it here